src/support_sphere/lib/presentation/pages/main_app/resource/resource_main_body.dart

```diff
@@ -148,6 +149,8 @@ class AllResourcesTab extends StatelessWidget {
             child: Text("No resources found"),
```

Comment:
This isn't in this PR but this should be in the string catalong

Reply:
Done https://github.com/uw-ssec/post-disaster-comms/pull/225/commits/9225cd1d827eadaac5b2b42f20bd32c25840f635